src/main/java/trackr/model/menu/ItemCost.java

```diff
@@ -0,0 +1,57 @@
+package trackr.model.menu;
+
+import static java.util.Objects.requireNonNull;
+import static trackr.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Item's cost in the Item list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidCost(String)}
+ */
+public class ItemCost {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Cost should only contain numbers, and it should be at most 4 digits long";
+    public static final String VALIDATION_REGEX = "^([1-9]{0,4})$";
```

Comment:
For the validation regex, use the following instead for `ItemCost`, `ItemPrice` and `ItemProfit`:
`    public static final String VALIDATION_REGEX = "^-?\\d*\\.\\d{2}$”;`

This supports numbers with 2 decimal places. Might want to check for `0.00` values too, whether it should be a valid cost/ price

Reply:
Thank you, I will send another PR after merging this PR.